lib/driver/pg.js

```diff
@@ -290,8 +290,18 @@ var PgDriver = Base.extend({
         } else {
           sql = util.format('ALTER TABLE "%s" ALTER COLUMN "%s" DROP DEFAULT', tableName, columnName);
         }
+        return this.runSql(sql).then(
+          setType.bind(this)
```

Comment:
I'm a little confused about your wording, which promise won't resolve? There are promises made by runSql in both `setDefaultValue` and `setType`. Additionally, `setType.bind(this)` is a promise itself too that passes it's results to nodeify(callback)

Am I missing something?